components/resources/brave_components_strings.grd

```diff
@@ -652,7 +652,7 @@
       <message name="IDS_BRAVE_UI_AMOUNT" desc="">Amount</message>
       <message name="IDS_BRAVE_UI_BRAVE_CONTRIBUTE_TITLE" desc="">Auto-Contribute</message>
       <message name="IDS_BRAVE_UI_BRAVE_CONTRIBUTE_TRANSACTION" desc="">Auto-Contribute [[processor]]</message>
-      <message name="IDS_BRAVE_UI_BRAVE_REWARDS" desc="">Brave Rewards</message>
+      <message name="IDS_BRAVE_UI_BRAVE_REWARDS" desc="">Rewards</message>
```

Comment:
Do we know that this string is definitely only used by the toolbar? It's pretty generic, so it may be reused elsewhere. cc: @zenparsing

Reply:
Yeah you're right, it's being used elsewhere. I might need to roll that back.

Reply:
I think you can just clone this string and give it a toolbar-specific name.